maritime-ai-service/scripts/deploy/.env.production.template

```
@@ -209,6 +218,8 @@ RATE_LIMIT_WINDOW_SECONDS=60
 # ─────────────────────────────────────────────────
 # SEMANTIC MEMORY
 # ─────────────────────────────────────────────────
+# Required while EMBEDDING_MODEL uses Gemini embeddings.
+GOOGLE_API_KEY=CHANGE_ME_your_gemini_api_key
 EMBEDDING_MODEL=models/gemini-embedding-001
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMBEDDING_MODEL key should go before the GOOGLE_API_KEY key

(UnorderedKey)